jekyll/_cci2/deploying-ios.md

```diff
@@ -196,11 +270,88 @@ desc "Upload to VS App Center"
     # Set up the required information to upload the
     # app binary to VS App Center
     appcenter_upload(
-      api_token: "YOUR_API_TOKEN",
+      api_token: ENV[VS_API_TOKEN],
       owner_name: "YOUR_VS_APPCENTER_USERNAME",
       owner_type: "user",
       app_name: "HelloWorld"
     )
   end
 end
+```
+
+## Uploading to TestFairy
+
+![TestFairy preferences image](  {{ site.baseurl }}/assets/img/docs/testfairy-open-preferences.png)
+
+1. On the TestFairy dashboard, navigate to the Preferences page.
+2. On the Preferences page, go to the API Key section and copy your API Key.
+3. Go to your project settings in CircleCI and create a new environment variable named `TESTFAIRY_API_KEY` with the value of the API Key.
+
+### Using Fastlane
+
+To configure uploading to TestFairy within Fastlane, see the following example:
+
+```ruby
+# fastlane/Fastfile
+default_platform :ios
+
+platform :ios do
+  before_all do
+    setup_circle_ci
+  end
+
+desc "Upload to TestFairy"
+  lane :upload_testfairy do
+    # Here we are using the CircleCI job number
+    # for the build number
+    increment_build_number(
+      build_number: "$CIRCLE_BUILD_NUM"
+    )
+    # Set up Adhoc code signing and build  the app
+    match(type: "adhoc")
+    gym(scheme: "HelloWorld")
+    # Set up the required information to upload the
+    # app binary to VS App Center
+    testfairy(
+      api_key: ENV[TESTFAIRY_API_KEY],
+      ipa: 'path/to/ipafile.ipa',
+      comment: ENV[CIRCLE_BUILD_URL]
+    )
+  end
+end
+```
+
+### Using curl
+
+Alternatively, you can upload binaries to TestFairy with `curl`
+
+```yaml
+# .circleci/config.yml
+version: 2.1
+jobs:
+  build:
+    #  insert build code here...
+  deploy:
+    steps:
+      - checkout
```

Comment:
you will need to use a workspace here to share the artifact from `build` to `deploy`. And you need to specify an executor in the `deploy` job.

Reply:
@marcomorain Good point, I just grabbed this code from the previous deployment docs. Will resolve this tomorrow with a better example config